cfgov/research_reports/management/commands/parse_research_report.py

```diff
@@ -24,6 +25,17 @@ def document_as_beautifulsoup(report_page):
     return BeautifulSoup(output, 'html.parser')
 
 
+def save_page(report_page):
+    print(" *** saving the page... ***")
+    report_page.process_report = False
+    try:
+        bot_user = User.objects.get(username='bot.user')
```

Comment:
This doesn't really affect us because we don't customize the Django user model, but I've read that best practice is to do something like [`User = get_user_model()`](https://docs.djangoproject.com/en/2.2/topics/auth/customizing/#referencing-the-user-model) instead of directly importing `User`. We do this in a bunch of places, [for example](https://github.com/cfpb/cfgov-refresh/blob/4d41e1ff660fee379b99d2a1099a5f7732e7b0fd/cfgov/core/tests/test_management_inactive_users.py#L25).

Reply:
Thanks for clearing that up! I added a commit to make this change.